descwl/analysis.py

```diff
@@ -705,20 +950,36 @@ def finalize(self,verbose,trace):
                 # assuming that we are in the sky-dominated limit.
                 data['snr_sky'][galaxy] = np.sqrt(np.sum(signal.array**2)/sky)
                 # Calculate this galaxy's SNR in various ways.
-                base = index*num_slices
+                base = index*6 #number of partials, better way to do this???
```

Comment:
Is there already an array with some dimension sized to the number of partials you could use to avoid hardcoding this number?